### PR TITLE
Panel Design Refresh

### LINKED ISSUE #29: [UI] Improve Devtools header section
I'm not a big fan of the 'Alpine.js is ready' heading at the top of the dev tools. If anything, it's wasted space.

I'm suggesting we move everything up and add a cleaner title bar to the top that would maybe output the Alpine version if we can find it (maybe using the script src) along with other information like the number of components, etc.

### PR DESCRIPTION
**Pull request by stephenoldham**
Hey all, I hope you don't mind, I've put a bit of time into a little panel refresh. I've tried to stick fairly close to the initial markup but made a few changes here and there. There's definite room for improvement, but I don't want to tread on anyones toes. 

**The Highlights**

- Design update (using Vue dev tools as a reference point)
Screenshot 2020-11-17 at 15 34 13
Screenshot 2020-11-17 at 15 39 45

- Improved checkbox styles and removed the need to click edit to reveal checkbox
- Added a fix for attributes edit value not being updated in state
- Updated style of attributes based on data type
- Added length for Array attributes i.e. Array[4]
- Lowercased html tags when used as component name
- Initially "empty" state before component is selected
- Added version detection (hopefully in the right way, I'll defer to you!) to display in the header
- Updated TailwindCss and UI plugin packages

(A nice potential improvement to implement, not already included above, would be resizeable panes using Split.js or similar)

I hope it's a helpful pull request and you're into the refresh! There's a lot that could be done but like I said I don't want to tread on anyones toes. This is me throwing my hat into the ring to be involved in the project. It's already a helpful tool for my day to day dev. See what you think 👍🏼

Potentially resolves #29

### CODE REVIEW

**Comment by ryangjchandler** [comment]
I don't actually use the DevTools much, but I know a lot of other people do still. @HugoDF I quite like this, what do you think?

**Comment by HugoDF** [comment]
Let's do it, if we merge it I can publish it.

Thanks for the time & effort 🙂

There's still that annoying issue where the devtools don't reload on page refresh that I never got to the bottom of.

File: tailwind.config.js [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+  		'sm': '.83333rem',
+      'base': '1rem',
+      'lg': '1.125rem',
+      'xl': '1.25rem',
+      '2xl': '1.5rem',
+  		'3xl': '1.875rem',
+  		'4xl': '2.25rem',
+      '5xl': '3rem',
+      '6xl': '4rem',
+    },
```
Comment (HugoDF) [inline tailwind.config.js:14]:
```suggestion
  	fontSize: {
        'xs': '.75rem',
  	    'sm': '.83333rem',
        'base': '1rem',
        'lg': '1.125rem',
        'xl': '1.25rem',
        '2xl': '1.5rem',
        '3xl': '1.875rem',
  		'4xl': '2.25rem',
        '5xl': '3rem',
        '6xl': '4rem',
    },
```

File: packages/shell-chrome/src/devtools-background.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ function createPanelIfHasAlpine() {
         "panel.html",
         panel => {
           // panel loaded
+          console.log(panel)
```
Comment (HugoDF) [inline packages/shell-chrome/src/devtools-background.js:34]: we probably don't need this

File: packages/shell-chrome/assets/panel.html [hunk 3]
```diff
@@ -30,103 +29,238 @@
...
+
+    <div class="bg-white flex flex-col relative h-screen w-full mx-auto">
+        <div 
+        id="header"
+        x-data="{
+            version: '',
+            latest: '2.7.3',
+
+            get isLatest(){
+                return this.version == this.latest
```
Comment (HugoDF) [inline packages/shell-chrome/assets/panel.html:42]:
```suggestion
                return this.version === this.latest
```

File: packages/shell-chrome/src/state.js [hunk 4]
```diff
@@ -125,15 +130,22 @@ export default class State {
     }
 
     updateXdata() {
+        document.getElementById("header").setAttribute(
+            "x-data",
+            `{
+                version : '${this.version}',
```
Comment (HugoDF) [inline packages/shell-chrome/src/state.js:136]: when we run this and update the attribute, what happens to the functions that were defined in x-data?

File: packages/shell-chrome/src/devtools-background.js (hunk 2 above)
Comment (stephenoldham) [reply packages/shell-chrome/src/devtools-background.js:34]: Oops! Sorry left in by mistake 😉

File: packages/shell-chrome/src/state.js (hunk 4 above)
Comment (stephenoldham) [reply packages/shell-chrome/src/state.js:136]: Yikes, good point. Could inject it directly into the alpine instance?
```suggestion
        let headerData = document.getElementById("header").__x.$data;
        headerData.version = this.version.detected;
        headerData.latest = this.version.latest;
```

Comment (HugoDF) [reply packages/shell-chrome/src/state.js:136]: yeah that could work

Comment (stephenoldham) [reply packages/shell-chrome/src/state.js]: Sorted, also updated the components to do the same 👍🏼

**Comment by stephenoldham** [comment]
Should be good to go now 👍🏼

File: tailwind.config.js [hunk 5]
```diff
@@ -1,8 +1,23 @@
...
+        'xs': '.75rem',
+  	    'sm': '.83333rem',
+        'base': '1rem',
+        'lg': '1.125rem',
+        'xl': '1.25rem',
+        '2xl': '1.5rem',
+        '3xl': '1.875rem',
+  	    '4xl': '2.25rem',
+        '5xl': '3rem',
+        '6xl': '4rem',
```
Comment (HugoDF) [inline tailwind.config.js:13]:
```suggestion
    fontSize: {
        'xs': '.75rem',
        'sm': '.83333rem',
        'base': '1rem',
        'lg': '1.125rem',
        'xl': '1.25rem',
        '2xl': '1.5rem',
        '3xl': '1.875rem',
        '4xl': '2.25rem',
        '5xl': '3rem',
        '6xl': '4rem',
```

**Review by HugoDF — COMMENTED** [review]
last one

**Comment by HugoDF** [comment]
Closes #29 

Closes #3 (the issue is stale, this design is much better)

File: packages/shell-chrome/assets/panel.html [hunk 6]
```diff
@@ -30,103 +29,237 @@
...
-                                                d="M10 14l2-2m0 0l2-2m-2 2l-2-2m2 2l2 2m7-2a9 9 0 11-18 0 9 9 0 0118 0z">
-                                            </path>
-                                        </svg>
-                                    </div>
-                                </span>
-                            </div>
-                        </template>
+            <div class="flex items-center space-x-2">
+                <a href="https://github.com/alpinejs/alpine" target="_blank">
+                    <svg class="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><title>AlpineJs Docs</title><path fill-rule="evenodd" d="M4 4a2 2 0 012-2h4.586A2 2 0 0112 2.586L15.414 6A2 2 0 0116 7.414V16a2 2 0 01-2 2H6a2 2 0 01-2-2V4zm2 6a1 1 0 011-1h6a1 1 0 110 2H7a1 1 0 01-1-1zm1 3a1 1 0 100 2h6a1 1 0 100-2H7z" clip-rule="evenodd"></path></svg>
```
Comment (HugoDF) [inline packages/shell-chrome/assets/panel.html:75]:
```suggestion
                    <svg class="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><title>Alpine.js Docs</title><path fill-rule="evenodd" d="M4 4a2 2 0 012-2h4.586A2 2 0 0112 2.586L15.414 6A2 2 0 0116 7.414V16a2 2 0 01-2 2H6a2 2 0 01-2-2V4zm2 6a1 1 0 011-1h6a1 1 0 110 2H7a1 1 0 01-1-1zm1 3a1 1 0 100 2h6a1 1 0 100-2H7z" clip-rule="evenodd"></path></svg>
```

**Pull request merged**